package.json

```diff
@@ -25,6 +25,7 @@
     "ember-cli": "2.8.0",
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-babel": "^5.1.6",
+    "ember-cli-datepicker": "2.0.1",
```

Comment:
Does this cli allow us to access the user's data? whatever they choose for their date range?